### DISCUSSION TITLE
Adding the remaining fair values in the module

### DISCUSSION THREAD

**Discussion by HadaRoshan** [synthetichealth/synthea#1392] [Q&A]
The module builder only provides you to give one or two fair fields, Is there any way where I can modify the other keys as well?
for example: I want to add some specific identifier to the patients for encounters but encounter module builder only provides you the option to add specific codes not the other fhir fields, Is there any solution for this?

**Comment by jawalonoski** [comment]
I am assuming you mean "FHIR" values, and not "fair" values.

Right now, the FHIR Exporters do what they do, subject to some configuration settings.

There is a feature under development called the "Flexporter" that may or may not address your concerns. See https://github.com/synthetichealth/synthea/pull/1219
